bibliography.bib

```diff
@@ -345,6 +345,37 @@ @Article{clusterdata:Mishra2010
 use the traces as inputs, e.g., in simulations or load predictions.
 Order: most recent first.
 
+################ 2016
+
+@INPROCEEDINGS{clusterdata:Sliwko2016,
+  title = {AGOCS – Accurate Google Cloud Simulator Framework},
```

Comment:
Thanks so much for doing this!

One nit: the capitalization needs protecting from latex:

title = {{AGOCS} – Accurate {Google} Cloud Simulator Framework},